### PR TITLE
docs: improve create-contentful-app readme [EXT-3559]

### PR DESCRIPTION
**Pull request by andipaetzold**
I'll copy the README to `packages/create-contentful-app/README.md` when it's approved

Preview: https://github.com/contentful/create-contentful-app/blob/120dc4e07b86f401701fe9d31cfa67fb320964bb/packages/contentful--create-contentful-app/README.md

### CODE REVIEW

File: packages/contentful--create-contentful-app/README.md [hunk 1]
```diff
@@ -1,19 +1,87 @@
...
 ```
-npx @contentful/create-contentful-app my-first-app
-cd my-first-app
-npm start
+
+## Package Manager
+
+`--npm` & `--yarn`
+
+Use npm or Yarn to manage dependencies. If omitted, defaults to the manager used to run `create-contentful-app`.
```
Comment (loweisz) [inline packages/contentful--create-contentful-app/README.md:41]: Any reason why yarn is capitalized and npm not?

Comment (andipaetzold) [reply packages/contentful--create-contentful-app/README.md:41]: That's the official capitalization

File: packages/contentful--create-contentful-app/README.md [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
+
+Both flags are mutually exclusive.
+
+## Template
+
+Select between predefined and custom templates:
+
+- `-js, --javascript`: Use default JavaScript template
+- `-ts, --typescript`: Use default TypeScript template
```
[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/contentful--create-contentful-app/README.md:50]: That's what `--help` says. Also confused me... gonna update

File: packages/contentful--create-contentful-app/README.md [hunk 3]
```diff
@@ -1,19 +1,87 @@
...
+# npm
+npm init contentful-app <app-name>
 
+# Yarn
+yarn create contentful-app <app-name>
 ```
-npx @contentful/create-contentful-app my-first-app
-cd my-first-app
-npm start
+
```
[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/contentful--create-contentful-app/README.md:36]: I'll add a subheader

File: packages/contentful--create-contentful-app/README.md [hunk 4]
```diff
@@ -1,19 +1,89 @@
...
+```bash
+npx create-contentful-app my-first-app
+```
+
+![Screenshot of `npx create-contentful-app my-app`](./docs/screenshot.png)
+
+## Bootstrap
+
+You can run `create-contentful-app` using one of the bollowing commands:
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/README.md:24]:
```suggestion
You can run `create-contentful-app` using one of the following commands:
```

File: packages/contentful--create-contentful-app/README.md [hunk 5]
```diff
@@ -1,19 +1,89 @@
...
+
+# Yarn
+yarn create contentful-app <app-name>
+```
+
+## CLI Options
+
+### Package Manager
 
+`--npm` & `--yarn`
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/README.md:41]:
```suggestion
`--npm` or `--yarn`
```

File: packages/contentful--create-contentful-app/README.md [hunk 6]
```diff
@@ -1,19 +1,89 @@
...
+- `-e, --example <example-name>`: Select a predefined template from https://github.com/contentful/apps/tree/master/examples
+- `-s, --source <url>`: Use a custom template. Format: URL (HTTPS or SSH) or vendor:user/repo (e.g., github:user/repo)
+
+These flags are mutually exclusive. If no flag is provided, the TypeScript template is used.
+
+### Help
+
+`--help`
+
+Shows all available CLI options
```
Comment (colomolo) [inline packages/contentful--create-contentful-app/README.md:62]:
```suggestion
Shows all available CLI options:
```

**Review by colomolo — APPROVED** [review]
lgtm

**Pull request merged**